source/14-4-0_Outcomes_Report_Overview.markdown

```diff
@@ -6,6 +6,11 @@
 
 The {{outcome}}s {{report}} summarises the changes experienced over time by your {{user}}s, and recorded in the {{outcome}}s tab. When a {{report}} is run Lamplight amalgamates the responses for each {{outcome}} measure. You can use filters to narrow your {{report}} down, for example choosing one specific measure, who is being reported on, or the dates you are interested in. For more on this, see [14.1.1 {{Report}} Filters](/help/index/p/14.1.1).
```

Comment:
The outcomes report summarises the changes experienced over time by your users**, and** recorded in the outcomes tab.
**DELETE**
When a report is run**,** Lamplight amalgamates the responses for each outcome measure.
**ADD COMMA**